graylog2-server/src/main/java/org/graylog2/shared/bindings/GenericBindings.java

```diff
@@ -64,7 +63,7 @@ protected void configure() {
 
         bind(ServiceManager.class).toProvider(ServiceManagerProvider.class).asEagerSingleton();
 
-        bind(HashedWheelTimer.class).toInstance(new HashedWheelTimer());
+        // TODO: Dedicated scheduled executor
```

Comment:
Is this a todo for this PR or a "nice to have"?

Reply:
It's a nice to have for `ThroughputCounter`.